taichi/ui/utils/utils.h

```diff
@@ -90,6 +92,12 @@ inline GLFWwindow *create_glfw_window_(const std::string &name,
     printf("GLFW reports no Vulkan support\n");
   }
 
+  // Reset the window hints to default
+  glfwDefaultWindowHints();
```

Comment:
Why?

Reply:
It seems that the Window we create should be invisible before we set the position.